development/webpack/ext/ChromeExtensionV3ViolationPlugin.js

```
@@ -9,22 +11,22 @@ class ChromeExtensionV3ViolationPlugin {
           });
+          if (hasFileChanged) {
+            compilation.assets[file] = new RawSource(content);
+          }
         });
       },
     );
   }
 }
 
-module.exports = ChromeExtensionV3ViolationPlugin;
+module.exports = ChromeExtensionV3ViolationPlugin;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)